### ISSUE TITLE
Log panel: Using a expr filter will show label "no unique labels"

### ISSUE THREAD

**Issue by negbie** [grafana/grafana#21769]
**What happened**:
Using the filter expr |="redis" will show the label "no unique labels"
**What you expected to happen**:
See the actual labels like when I don't use the filter expr |="redis"
**How to reproduce it (as minimally and precisely as possible)**:
Use a filter expr in Log Panel
**Anything else we need to know?**:

**Environment**:
- Grafana version:
latest (6.6.0)
- Data source type & version:
dockerhub
- OS Grafana is installed on:
debian 10
- User OS & Browser:
latest ubuntu
- Grafana plugins:
none
![logpanelfilterbug](https://user-images.githubusercontent.com/20154956/73202165-c3605380-413a-11ea-971a-ac49177b7210.jpeg)

- Others:

**Comment by negbie** [comment]
Log panel query is:
{hostname=~"${hostname:pipe}", level=~"${level:pipe}", program=~"${program:pipe}"} $search

**Comment by negbie** [comment]
Hmm just noticed that there are cases where I will see the label and using a filter expression.

**Comment by ivanahuckova** [comment]
Hello negbie, I have tried to reproduce the issue, but I wasn't able to. Is it possible that those logs just don't have unique labels. Common labels are not displayed in that column as it is dedicated to unique labels only.

**Comment by negbie** [comment]
@ivanahuckova thank you for looking into this. Those logs have unique labels but something strange is going on. Today I used the same filter expression and I see the labels

![image](https://user-images.githubusercontent.com/20154956/73261934-606bcc80-41cd-11ea-81c0-24f8ed4dd172.png)

Maybe it's related to a loki restart since I did a docker-compose pull and saw that grafana and loki image were updated.
